paper/paper.md

```diff
@@ -0,0 +1,77 @@
+    orcid: 0000-0001-7419-8978
+    corresponding: true
+    affiliation: 1
+affiliations:
+ - name: IMASL-CONICET. Universidad Nacional de San Luis. San Luis, Argentina
+   index: 1
+ - name: Independent Researcher, Spain
+   index: 2
+ - name: Department of Computer Science, University of Helsinki, Finland
+   index: 3
+date: 31 January 2023
+bibliography: paper.bib
+
+---
+
+# Summary
+
+In a Bayesian modeling workflow, a prior distribution can be choosen in different ways as long as it captures the uncertainty about model parameters prior to observing any data. Particularly, prior elicitation refers to the process of transforming the knowledge of a particular domain into well-defined probability distributions. Here we introduce PreliZ, a Python package aimed at helping practitioners choose prior distributions.
+
+# Statement of need
+
+Specifying useful priors is a central aspect of Bayesian statistics `[@gelman:2013, @martin:2021, @martin:2022]`, yet prior elicitation techniques that would make this step easier and more systematic are not routinely used within practical Bayesian workflows `[@mikkola:2021]`. Instead, practitioners tipically rely on ad hoc procedures based on a mix of their own experience and recommendations available in the literature, which in general has not been systematized `[@sarma:2020]`. One reason is that current solutions are simply not sufficient for practical data analysis and do not integrate well with probabilisitc programming libraries `[@mikkola:2021]`. PreliZ is a library for prior elicitation that aims to facilitate the task for practitioners by offering a set of tools for the various facets of prior elicitation. It covers a range of methods, from unidimensional prior elicitation on the parameter space to predictive elicitation on the observed space. The goal is to be compatible with probabilistic programming languages in the Python ecosystem like PyMC and PyStan, which involves including the distributions and their parameterization that are supported by these languages.
+
+# Software API and Features
+
+PreliZ provides functionality for four core aspects: (1) Specification and manipulation of prior distributions, (2) Visualization of priors and the induced predictive distributions, (3) User interaction components, and (4) Algorithms learning priors from user-provided information.
+PreliZ distributions are built on top of SciPy distributions, providing easy access to all standard functionality, in particular random sampling and quantiles needed in the elicitation process. However, PreliZ extends many of the distributions for increased user-friendliness. For instance, PreliZ uses a canonical parametrization, instead of loc-scale style used by many of the SciPy distributions, and adds common alternative parametrizations,  e.g. Beta-distribution also in terms of mean and standard deviation. Finally, PreliZ also includes some distributions not available in SciPy, like the HalfStudent, or LogitNormal.
```

Comment:
Just realized this is `HalfStudent` instead of `HalfStudentT` (also on the library). It would probably be best to update both Student and HalfStudent to StudentT/HalfStudentT following pymc (would match name exactly) and stan (same name but in stan distributions use snake case instead of CamelCase)

Reply:
Solved in #219 on the library side. We just need to update the draft to match